test/python/reservoir_mpi_test.py

```diff
@@ -337,13 +337,12 @@
 i=10   
 timestep = str(i).rjust(5, '0')
 
-sig_digits = 4
-abs_value = 1e-12
-
```

Comment:
These are the tolerance settings in question. I believe the main issue is using too few significant digits